relativedeltafield/__init__.py

```diff
@@ -87,13 +148,15 @@ class RelativeDeltaField(models.Field):
 					 "ISO8601 interval format.")
 	}
 	description = _("RelativeDelta")
-
+	form_class = RelativeDeltaFormField
+	descriptor_class = RelativeDeltaDescriptor
 
 	def db_type(self, connection):
-		if connection.vendor == 'postgresql':
+		if connection.vendor == 'postgresql3':
```

Comment:
Why is this needed, and won't this break stuff on my own installations where this worked?

Reply:
This was a mistake fixed in a subsequent commit.